### DISCUSSION TITLE
People

### DISCUSSION THREAD

**Discussion by agzam** [Emacs-SF/meetup-notes#14] [Ideas]
I added "[Backlog](https://github.com/Emacs-SF/meetup-notes/wiki/Backlog)" wiki page maybe it can work as our "piggy bank" for things we want to discuss at future meetings. 

Similarly, I wanted to suggest adding a "People" page - a list of interesting individuals - authors, maintainers, experts. So when the next time, let's say we want to discuss a topic, we'd try to invite someone with a deep understanding of the subject and maybe they'd share some tips. But then that may feel like verging into "creepy" territory, borderline with invading personal space.

What do think?

**Comment by wrightmikea** [comment]
If we limit our creating of  your proposed People page entries to the person's "preferred" public "landing page" it shouldn't be considered creepy.  For example, an emacs package creator's github profile page.  If they want more contact info public (that isn't already on their github page) they can edit our wiki page (or they can delete themselves from our wiki page if they so desire).   If our People page entry refers to a blogger, then we could use their blog profile link, or if their web presence in primarily Twitter, that page, etc.

**Comment by jefftrull** [comment]
Great session ideas, Ag!
